XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="MSBuild.Sdk.Extras/2.0.54">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;netstandard2.0;netstandard2.1;Xamarin.iOS10;MonoAndroid90;MonoAndroid10.0;Xamarin.TVOS10;Xamarin.WatchOS10;Xamarin.Mac20;tizen40</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;Xamarin.iOS10;MonoAndroid90;MonoAndroid10.0;Xamarin.TVOS10;Xamarin.WatchOS10;Xamarin.Mac20;tizen40</TargetFrameworks>
```

Comment:
I am not sure we want to drop support of ```netstandard1.0```
CC @jfversluis

Reply:
netstandard1.0 doesn't support api: CultureInfo.Text.IsRightToLeft

Reply:
let's retrun it back as well)

Reply:
The file is outdated. I rolled back this changes